src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -85,6 +85,10 @@ public string ClientVersion
 
         public bool LegacyCacheCompatibilityEnabled { get; internal set; } = true;
 
+        #region Region
+        public string AzureRegion { get; set; }
```

Comment:
>set [](start = 41, length = 3)

internal set ? - see a few other params which are not internal set, given the class is internal not important however for consistency?

Reply:
Made the rest "public". As you say, it's just a consistency issue.